src/server/dfly_main.cc

```diff
@@ -701,6 +702,24 @@ void PrintBasicUsageInfo() {
   std::cout << endl;
 }
 
+void ParseFlagsFromEnv() {
+  const auto& flags = absl::GetAllFlags();
+  for (const auto& flag_entry : flags) {
+    auto& flag_name = flag_entry.first;
+    auto& flag = flag_entry.second;
+    const char* flag_env_value = getenv(absl::StrCat("FLAGS_", flag_name).data());
```

Comment:
Use c_str if you want a c string. Afaik data = c_str on all implementations but its theoretically not guaranteed

Reply:
Ah, actually

> In C++11 the null termination guarantee is specified so string.data()==string.c_str() is always valid.

The standard library is a mess 😂